src/main/python/hutts_utils/image_handling.py

```diff
@@ -0,0 +1,76 @@
+import cv2
+import base64
+import numpy as np
+from image_processing.sample_extract import FaceExtractor
+from imutils.convenience import url_to_image
+from flask import jsonify, make_response
+from hutts_utils.hutts_logger import logger
+
+
+def grab_image(path=None, stream=None, url=None):
+    """
+    This function grabs the image from URL, or image path and applies necessary changes to the grabbed
+    images so that the image is compatible with OpenCV operation.
+    Author(s):
+        Stephan Nell
+    Args:
+        path (str): The path to the image if it reside on disk
+        stream (str): A stream of text representing where on the internet the image resides
+        url(str): Url representing a path to where an image should be fetched.
+    Returns:
+        (:obj:'OpenCV image'): Image that is now compatible with OpenCV operations
+    TODO:
+        Return a Json error indicating file not found
+    """
+    # If the path is not None, then load the image from disk. Example: payload = {"image": open("id.jpg", "rb")}
+    if path is not None:
+        logger.debug("Grabbing from Disk")
+        image = cv2.imread(path)
+    # otherwise, the image does not reside on disk
+    else:
+        # if the URL is not None, then download the image
+        if url is not None:
+            logger.debug("Downloading image from URL")
+            return url_to_image(url)
+        # if the stream is not None, then the image has been uploaded
+        elif stream is not None:
+            # Example: "http://www.pyimagesearch.com/wp-content/uploads/2015/05/obama.jpg"
+            logger.debug("Downloading from image stream")
+            data = stream.read()
+            image = np.asarray(bytearray(data), dtype="uint8")
+            image = cv2.imdecode(image, cv2.IMREAD_COLOR)
+    return image
+
+
+def face_extraction_response(image, text_extract_result=None):
```

Comment:
Why did you move this function here? It seems as if it was only used in one of the modules. This function also introduces a dependency on one of the other packages (`image_processing`) which defeats the purpose of a utility function. A utility module should not make use of any other module that may or may not make use of its functions. I am happy with the moving of the `grab_face` function, however I do disagree with the decision to move this function here. What are your thoughts on this?

Reply:
Okay yeah, I guess it not necessary, first thought was that it might be used in verify controller since not everything is implemented there yet but does not seem it will be used. I'll quickly move it.